src/taco-cli/cli/utils/checkForNewerVersion.ts

```diff
@@ -56,7 +56,7 @@ class CheckForNewerVersion {
 
     public showOnExitAndIgnoreFailures(): void {
         try {
-            this.showOnExit().catch(error => { }).done();
+            this.showOnExit().catch(error => tacoUtility.UtilHelper.emptyMethod(error)).done();
```

Comment:
Why not just `this.showOnExit().catch(tacoUtility.UtilHelper.emptyMethod).done();` instead?